### PR TITLE
Create HubSpot drop

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

**Comment by jimaek** [comment]
The CSS files are not required?

**Comment by adamschwartz** [comment]
Yeah probably each of these should be included:

```
https://github.com/HubSpot/drop/tree/master/dist/css
https://github.com/HubSpot/drop/blob/master/dist/css/drop-theme-basic.css
https://github.com/HubSpot/drop/blob/master/dist/css/drop-theme-arrows.css
https://github.com/HubSpot/drop/blob/master/dist/css/drop-theme-arrows-bounce.css
https://github.com/HubSpot/drop/blob/master/dist/css/drop-theme-arrows-bounce-dark.css
https://github.com/HubSpot/drop/blob/master/dist/css/drop-theme-twipsy.css
https://github.com/HubSpot/drop/blob/master/dist/css/drop-theme-hubspot-popovers.css
```

File: files/drop/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "drop",
+  "repo": "HubSpot/drop",
+  "files": {
+    "include": ["dist/js/drop.min.js"]
```
Comment (jimaek) [inline files/drop/update.json]: Add `"basePath": "dist/",`
And then include`./js/*.min.*` and `./css/*.min.*`

**Comment by dandv** [comment]
Hey @jimaek, I've submitted another commit to include the extra files.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by dandv** [comment]
That's a nice bot :)

**Comment by jimaek** [comment]
Automation is awesome :)

**Pull request closed without merging**